src/main/java/org/commcare/cases/instance/StorageBackedChildElement.java

```diff
@@ -227,4 +254,6 @@ protected TreeElement cache() {
     }
 
     protected abstract TreeElement cache(QueryContext context);
```

Comment:
It will eventually, it just involves a fair amount more testing, and the meta data semantics can be slightly more complex, so I'm holding off on that until I can test it out and see the impact/value.

Reply:
I'm not sure if adding a note will be more helpful than confusing. The new changes to cache() added to IndexedFixtureChildElement are a new optimization pattern entirely that just isn't  implemented in the other data models because it isn't as useful for them yet (similar to the way that the CaseChildElement is able to optimize index/* queries differently than the other data types).

I don't think it's specific technical debt to re-implement for cases and ledgers (IE: Nothing about the way they work is deprecated now), so I'll add a card to the performance board which mentions that it's low hanging fruit to re-implement this new pattern if it would be useful for optimizing something about those models later.